src/app/models/blocks.test.js

```diff
@@ -0,0 +1,137 @@
+import { videoBlock, rawVideoBlock, blockContainingText, blockBase, rawImageBlock, stdArrayModelBlock, blockArrayModel, imageBlock, rawImageModel } from './blocks';
+
+const testModel = {
+    type: "testingType",
+    model: "TheModel"
+}
+
+const rawBlock = (t) => ({
+    type: t,
+    model: testModel
+});
+
+const rawBlockTest = (type, blockFunction) =>(
+    describe('Raw Video Block', () => {
+        const raw = blockFunction(testModel);
+        const block = rawBlock(type);
+        test('takes a Raw Image Model and outputs the correct json', () => {
+            expect(raw).toEqual(block);
+        });
+    })
+);
+
+describe('Block Containing Text', () => {
+    test('yes', () => {
```

Comment:
I recommend updating the structure of the tests - to have fewer 'describe' blocks and instead of using the `test('explanation here', ()=> {});` format, to use `it('should render a block containing text', ()=>{}`. This will enhance readability of the tests and more clearly highlight the expected outcome of a test.